package.json

```diff
@@ -16,10 +16,17 @@
 	],
 	"bin": "dist/cli.mjs",
 	"scripts": {
+		"prepare": "simple-git-hooks",
```

Comment:
This installs the commit hooks to git whenever the dev runs `npm i` or `pnpm i`.